Solution/source/Menu/Routine.cpp

```diff
@@ -129,25 +140,11 @@ void ThreadMenyooMain()
 	MenyooMain();
 }
 
-void ThreadMenyooConfig()
+void TickMenyooConfig()
 {
-	//for (UINT8 configT = 0; MenuConfig::ConfigInit() < 0 && configT < 10; configT++)
-	//std::this_thread::sleep_for(std::chrono::seconds(1));
-
-	DWORD _programTick = 0U;
-
-	std::this_thread::sleep_for(std::chrono::seconds(9));
-	bool bConfigHasNotBeenRead = true;
-
-	for (;;)
+	if (GetTickCount() > g_MenyooConfigOnceTick + 9000)
```

Comment:
I'm not sure about this one but wouldn't if( _programTick % 9000 ) work? Cause in that case there's no need for the this variable 😄

Reply:
If the 9 second everything-blocking delay can be removed/ignored, the config tick can be used. (You'd still need one global, since this function is called on tick, no local variables here).